### DISCUSSION TITLE
Rename all files based on a particular format and separator

### DISCUSSION THREAD

**Discussion by lamyergeier** [stashapp/stash#5427] [Q&A] [answered]
By using tagger, I was able to update the scene title. Is it possible to rename all video files to the following format and separator `~~~`

${Studio}~~~`${Scene_Title}~~~${Video_Size_in_MB}~~~${Video_length}~~~${Video_bits_per_second}~~~${Video_Resolution}.${Extension}`

## For example:

Corbin_Fisher~~~Thad_takes_Cain~~~950MB~~~00:18:21~~~717736bps~~~1920x1080.mp4

**Comment by feederbox826** [comment, marked as answer]
Stash does not do renaming, this can be accomplished with one of the many "Renamer" plugins https://docs.stashapp.cc/plugins/list/#renamefile
